e2e-tests/trade-finance-manager/cypress/integration/journeys/facility/facilities.spec.js

```diff
@@ -0,0 +1,90 @@
+
+      cy.createFacilities(dealId, mockFacilities, MOCK_MAKER_TFM).then((createdFacilities) => {
+        dealFacilities.push(...createdFacilities);
+      });
+
+      cy.submitDeal(dealId, dealType);
+    });
+  });
+
+  beforeEach(() => {
+    cy.login(MOCK_USERS[0]);
+  });
+
+  after(() => {
+    cy.deleteTfmDeals(dealId);
+    dealFacilities.forEach((facility) => {
+      cy.deleteFacility(facility._id, MOCK_MAKER_TFM);
+    });
+  });
+
+  it('renders all facilities from TFM', () => {
+    cy.visit(relative('/facilities'));
+    cy.url().should('eq', relative('/facilities'));
+
+    page.facilitiesPage.ukefFacilityIdColumn().invoke('attr', 'aria-sort').should('contain', 'ascending');
+    page.facilitiesPage.productColumn().should('contain', 'Product');
+    page.facilitiesPage.dataTypeColumn().should('contain', 'Type');
+    page.facilitiesPage.exporterColumn().should('contain', 'Exporter');
+    page.facilitiesPage.valueColumn().should('contain', 'Value (export currency)');
+    page.facilitiesPage.coverEndDateColumn().should('contain', 'Cover end date');
+    page.facilitiesPage.tfmFacilitiesTable().find('.govuk-table__row').eq(1).find('[data-cy="facility__ukefFacilityId"]')
+      .should('contain', '10000000002');
+    page.facilitiesPage.tfmFacilitiesTable().find('.govuk-table__row').eq(1).find('[data-cy="facility__product"]')
+      .should('contain', 'BSS/EWCS');
+    page.facilitiesPage.tfmFacilitiesTable().find('.govuk-table__row').eq(1).find('[data-cy="facility__facilityType"]')
+      .should('contain', 'bond');
+    page.facilitiesPage.tfmFacilitiesTable().find('.govuk-table__row').eq(1).find('[data-cy="facility__companyName"]')
+      .should('contain', 'Mock company');
+    page.facilitiesPage.tfmFacilitiesTable().find('.govuk-table__row').eq(1).find('[data-cy="facility__facilityValue"]')
+      .should('contain', 'GBP 12,345.00');
+    page.facilitiesPage.tfmFacilitiesTable().find('.govuk-table__row').eq(1).find('[data-cy="facility__coverEndDate"]')
+      .should('contain', '20 Oct 2020');
+
+    page.facilitiesPage.tfmFacilitiesTable().find('.govuk-table__row').eq(2).find('[data-cy="facility__ukefFacilityId"]')
+      .should('contain', '10000000003');
+    page.facilitiesPage.tfmFacilitiesTable().find('.govuk-table__row').eq(2).find('[data-cy="facility__product"]')
+      .should('contain', 'BSS/EWCS');
+    page.facilitiesPage.tfmFacilitiesTable().find('.govuk-table__row').eq(2).find('[data-cy="facility__facilityType"]')
+      .should('contain', 'loan');
+    page.facilitiesPage.tfmFacilitiesTable().find('.govuk-table__row').eq(2).find('[data-cy="facility__companyName"]')
+      .should('contain', 'Mock company');
+    page.facilitiesPage.tfmFacilitiesTable().find('.govuk-table__row').eq(2).find('[data-cy="facility__facilityValue"]')
+      .should('contain', 'GBP 1,234.00');
+    page.facilitiesPage.tfmFacilitiesTable().find('.govuk-table__row').eq(2).find('[data-cy="facility__coverEndDate"]')
```

Comment:
Would be nice to do this so it's easier to read, not a blocker :)

```js
const row1 = page.facilitiesPage.tfmFacilitiesTable().find('.govuk-table__row').eq(1);
```

Reply:
I've updated this now. It's much easier to read now :)